--- memcached-1.4.33/configure.orig     2016-12-01 19:00:34.936331172 +0100
+++ memcached-1.4.33/configure  2016-12-01 19:00:56.182328104 +0100
@@ -6425,14 +6425,14 @@

 if test "$ICC" = "yes"
 then
-      CFLAGS="$CFLAGS -diag-disable 187 -Wall -Werror"
+      CFLAGS="$CFLAGS -diag-disable 187 -Wall"

 $as_echo "#define _GNU_SOURCE 1" >>confdefs.h

 elif test "$GCC" = "yes"
 then
   GCC_VERSION=`$CC -dumpversion`
-  CFLAGS="$CFLAGS -Wall -Werror -pedantic -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls"
+  CFLAGS="$CFLAGS -Wall -pedantic -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls"
   case $GCC_VERSION in
     4.4.*)
     CFLAGS="$CFLAGS -fno-strict-aliasing"