--- ghostscript-9.21/Makefile.in.orig
+++ ghostscript-9.21/Makefile.in
@@ -459,7 +459,7 @@
 # Solaris may need -lnsl -lsocket -lposix4.
 # (Libraries required by individual drivers are handled automatically.)
 
-EXTRALIBS=$(XTRALIBS) @LIBS@ @DYNAMIC_LIBS@ @FONTCONFIG_LIBS@ @FT_LIBS@ @JPX_AUTOCONF_LIBS@ @EXPAT_LIBS@
+EXTRALIBS=$(XTRALIBS) @LIBS@ @DYNAMIC_LIBS@ @FONTCONFIG_LIBS@ @FT_LIBS@ @JPX_AUTOCONF_LIBS@ @EXPAT_LIBS@ @LDFLAGS@
 AUXEXTRALIBS=@AUXEXTRALIBS@
 
 # Define the standard libraries to search at the end of linking.
